.github/workflows/update-attributions.yml

```diff
@@ -115,7 +115,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           # Use PAT to ensure that the commit later can trigger status check workflows
```

Comment:
This comment was meant to explain why we need an elevated privilege token for this step

Reply:
Indeed I am not sure How I missed this